crux_macros/src/bridge.rs

```diff
@@ -0,0 +1,107 @@
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::ItemStruct;
+
+pub fn bridge_impl(input: ItemStruct) -> TokenStream {
+    let ident = &input.ident;
+    quote! {
+        #[derive(Default)]
+        #input
+
+        static CORE: ::std::sync::LazyLock<::crux_core::bridge::Bridge<#ident>> =
+            ::std::sync::LazyLock::new(|| {
+                ::crux_core::bridge::Bridge::new(::crux_core::Core::new())
+            });
+
+        #[cfg(not(target_family = "wasm"))]
+        const _: () = assert!(
+            uniffi::check_compatible_version("0.29.2"),
```

Comment:
Elsewhere we do `::uniffi` - does this make the macro potentially leaky if someone redefines `uniffi`?

Reply:
thanks. I missed this. Done now.